README.md

```diff
@@ -46,21 +55,90 @@ GET /:path
 ### Flags
 
 | Name | Description                                  |
-|------|----------------------------------------------|
+| ---- | -------------------------------------------- |
 | -v   | Print info messages                          |
 | -vv  | Print more info messages                     |
 | -w   | Reload the contents when they change on disk |
 
 ### Docker
 
-If you have docker installed, you can also run the repo using `docker-compose up --build`
+If you have docker installed, you can also run the repo using `docker compose up --build`
+
+Make sure to copy `.env.example` to `.env` first, and populate `TOKEN` with you personal github token if needed (if your content repo is private).
```

Comment:
We could add that the token must be set but the value doesn't matter if the repo is public. We could also set it to something in `.env.example`.

Or we could make the token not be mandatory